src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -152,7 +152,7 @@ typedef struct {
   uint16_t queued_len;
   uint16_t max_packet_size;
   uint8_t ep_idx;   // index for USB_EPnR register
-  bool in_complete; // Workaround for ISO IN EP doesn't have interrupt mask
+  bool iso_in_sending; // Workaround for ISO IN EP doesn't have interrupt mask
```

Comment:
@HiFiPhile I have rename and invert logic for this flag. It now means if we have an active data for ISO IN, and limit its affect on ISO endpoint only. Let me know it makes sense to you.

Reply:
Yes prefect